web_src/less/_repository.less

```diff
@@ -1049,7 +1056,7 @@
           > .bottom.segment {
             background: #f3f4f5;
 
-            .ui.images::after {
+            .ui.images:after {
```

Comment:
This won't insert pseido element anymore...

Reply:
Should still work as the CSS parser accepts both single and dobule colons, but I guess if we want to be correct, we better convert to double colon and lint for it.

https://css-tricks.com/to-double-colon-or-not-do-double-colon/

Reply:
Both are correct but have different use cases